src/Raven.Studio/typescript/viewmodels/manage/adminLogsSettingsDialog.ts

```diff
@@ -0,0 +1,67 @@
+import dialog = require("plugins/dialog");
+import dialogViewModelBase = require("viewmodels/dialogViewModelBase");
+import getAdminLogsSettingsCommand = require("commands/maintenance/getAdminLogsSettingsCommand");
+import saveAdminLogsSettingsCommand = require("commands/maintenance/saveAdminLogsSettingsCommand");
+import genUtils = require("common/generalUtils");
+
+class adminLogsSettingsDialog extends dialogViewModelBase {
+
+    possibleLogModes: Array<Sparrow.Logging.LogMode> = ["Operations", "Information"];
+    currentServerMode = ko.observable<Sparrow.Logging.LogMode>();
+    selectedMode = ko.observable<Sparrow.Logging.LogMode>();
+    
+    fullPath = ko.observable<string>();
+    retentionTime = ko.observable<string>();
+    retentionSize = ko.observable<Sparrow.Size>();
+    compress: boolean;
+
+    retentionTimeText: KnockoutComputed<string>;
+    retentionSizeText: KnockoutComputed<Sparrow.Size>;
```

Comment:
are you sure type here should be sprarrow.size?

Reply:
Sparrow.Size is now being translated to 'number'